picturae_importer.py

```diff
@@ -27,11 +29,14 @@ class PicturaeImporter(Importer):
 
     def __init__(self, paths, date_string=None):
         super().__init__(picturae_config, "Botany")
+        self.logger = logging.getLogger("PicturaeImporter")
 
         self.init_all_vars(date_string=date_string, paths=paths)
 
         # running csv create
-        CsvCreatePicturae(date_string=self.date_use)
+        csv_create_picturae = CsvCreatePicturae(date_string=self.date_use, logger=self.logger)
```

Comment:
I am not sure if this is the right move. But I simply reused the same logger from the PicturaeImport class as a parameter in the CsvCreatePicturae class. My thinking being that CsvCreatePicturae is simply part of the same larger import process as PicturaeImport

Reply:
Hm - that's a good in person converation to have. Let's have you write up the business rules as we discussed, and then take some quality time to go over everything so i can understand your full flow.